src/App.js

```diff
@@ -11,13 +13,10 @@ function App() {
   const [imageUrl, setImageUrl] = useState(null);
 
   useEffect(() => {
-    fetch(
-      `https://api.unsplash.com/photos/random/?client_id=XqByhaz8qx069KZ_7Kk528zhO_02RDftaH3ioB8y1No&&orientation=landscape&&query=mountains%20dark`
-    )
-      .then((res) => res.json())
-      .then((x) => {
-        console.log(x);
-        setImageUrl(x.urls.regular);
+    fetch(UNSPLASH_API_URL)
+      .then((response) => response.json())
```

Comment:
whenever you write .then make sure u write .catch too, what if ur UNSPLASH_API_URL server is down?

Reply:
Thanks for pointing it out.